docs/lib/getting-started/fragments/android/add-api.md

```diff
@@ -1,42 +1,205 @@
+    ```
+
+    Sign in and press **Enter**.
+
+1. Next, specify the AWS Region you'd like to use and the name of the IAM user to create to manage Amplify resources:
+
+    ```console
+    Specify the AWS Region
+    ? region: 
+        us-east-1
+    Specify the username of the new IAM user:
+    ? user name:  (amplify)
+        amplify 
+    ```
+
+1. Amplify will open the AWS Management Console to the IAM Add User page. Review the **User name**, and then press **Next: Permissions**.
+
+1. The next page grants the user *AdministratorAccessor*. Press **Next: Tags**.
+
+1. Optionally add tags to the new user. When finished, press **Next: Review**.
+
+1. Review the new user details. When finished, press **Create user**.
+
+1. Go back to your terminal and press **Enter**. Next, you will be prompted for credentials for the new user. Copy and paste the credentials for this user when prompted and name your profile **Amplify**.
+
+    ```console
+    Enter the access key of the newly created user:
+    ? accessKeyId:  [hidden] 
+        ********************
+    ? secretAccessKey: [hidden]
+        ****************************************
+    ? Profile Name:  (default)
+        amplify
+
+    Successfully set up the new user.
+    ```
+
+1. In your terminal, ensure you are in the root of your project directory. Run `amplify init`. Enter a name for your environment such as *tutorial*. Select your preferred editor, and your AWS profile:
+
+    ```console
+    ? Enter a name for the environment
+        tutorial
+    ? Choose your default editor:
+        IntelliJ IDEA
+    ? Do you want to use an AWS profile?
+        Yes
+    ? Please choose the profile you want to use
+        amplify
+
+    Adding backend environment dev to AWS Amplify Console app: abcd1234efg567
+    ⠋ Initializing project in the cloud...
+    ```
+
+    This initializes your Amplify project on the backend
+
+1. In the terminal, run `amplify push`. You can select the defaults to the prompts:
+
+    ```console
+    ✔ Successfully pulled backend environment dev from the cloud.
+
+    Current Environment: dev
+
+    | Category | Resource name     | Operation | Provider plugin   |
+    | -------- | ----------------- | --------- | ----------------- |
+    | Api      | amplifyDatasource | Create    | awscloudformation |
+    
+    ? Are you sure you want to continue?
+        Yes
+    ? Do you want to generate code for your newly created GraphQL API
+        Yes
+    ? Enter the file name pattern of graphql queries, mutations and subscriptions
+        app/src/main/graphql/**/*.graphql
+    ? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions
+        Yes
+    ? Enter maximum statement depth [increase from default if your schema is deeply nested]
+        2
+        
+    ⠋ Updating resources in the cloud. This may take a few minutes...
+    ```
+
+1. Run your application. This will create a new Todo and synchronize it to your API.
+
```

Comment:
how will this create new todo's?  seems like we should talk about how it should synchronize anything that was previously stored locally to the backend.. 

Also, seems like we need a lead in sentence like.. "to verify that data has been sync'ed to the backend, one way is to use the appsync console.  to do this..."

Reply:
It's kind of funny. When I wrote this synchronization wasn't working, so I think I just didn't even think about this. Let me rework this.